packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.html

```diff
@@ -16,69 +16,90 @@
         >
         </lightning-button-icon>
         <template if:true={hasActiveValues}>
-          <button class="facet__clear-filter slds-button slds-grid slds-grid_vertical-align-center slds-var-p-top_x-small slds-var-p-horizontal_x-small" onclick={clearSelections}>
-            <lightning-icon class="slds-current-color slds-var-m-right_xx-small" icon-name="utility:close" alternative-text={altText} size="xx-small"></lightning-icon>
+          <button
+            class="facet__clear-filter slds-button slds-grid slds-grid_vertical-align-center slds-var-p-top_x-small slds-var-p-horizontal_x-small"
+            onclick={clearSelections}
+          >
+            <lightning-icon
+              class="slds-current-color slds-var-m-right_xx-small"
+              icon-name="utility:close"
+              alternative-text={altText}
```

Comment:
Instead of having a button that contains an icon which has alternative test, I think it would be better to have a button with an aria-label (which describes its action) and `aria-hidden="true"` on the icon.